mod/metrics/set/abstract/value.rb

```diff
@@ -24,6 +24,10 @@ def company_key
   company.to_name.key
 end
 
+def company_id
```

Comment:
maybe this should be called company_mark?

Reply:
The code is wrong, not the name. Looks like I hit "undo" one time too often when I removed what I did after I corrected this.